tools/terraform_sync_tool/qa/terraform-sync-tool/terragrunt.hcl

```diff
@@ -0,0 +1,40 @@
+terraform {
+  source = "../../modules/bq-setup"
+}
+
+include "root" {
+  path   = find_in_parent_folders()
+  expose = true
+}
+
+locals {
+  # TODO: Update your dataset ID
+  dataset_id = "YOUR_DATASET_ID"
```

Comment:
```suggestion
  dataset_id = "DatasetForTest"
```

Reply:
Since you call your test table TableForTest, this follows the same naming convention